OSG Tech meeting notes for 11/28/23 (#103)
osg-htc/technology: docs/meetings/2023/TechArea20231128.md

Remaining OSG 23 items <https://opensciencegrid.atlassian.net/browse/SOFTWARE-5503>.
-   Figure out if we need VOMS for EL9/an update to EL8

> -   AI (Tim): Update tarball scripts to use `23-main` (SOFTWARE-5713)
>     - Done in previous release. Changing from 'latest' releases to concious choice to change version, possibly in Febuary?

Reviewer: ```suggestion
```

Dropping this since the work was actually done. Dropping the `latest` symlink is orthogonal and more of a policy decision

### Support Update

None this week

> -    Mat: Kurt running into xrootd segfault, registry stuff (inturupted login flow, new site in Lebanon (stolen by Brian L)), Brain initiative passed to Brian B(?)

Reviewer: ```suggestion
-    JLab (Mat): Kurt running into another XRootD segfault, 
-    Topology/COManage (Mat): interrupted enrollment flow
-    Lebanon (BrianL): new site, consulting with management regarding policy
```

I like to organize these by site/service. Also, spelling

-   Ready for Testing
    -   XRootD 5.6.3-1.2
        -   Another bugfix incoming for XRootD
    -   osg-system-profiler 1.7.0
    -   htcondor-ce 23.0.1
    -   osg-vo-client (Ready by Thursday)
-   Ready for Release
    -   Nothing yet
 
### Discussion
-   CA cert updates? Punting for ~week.
-   RE: AI (Tim): Update tarball scripts to use `23-main` (SOFTWARE-5713)

>     -    Done in previous release. Changing from 'latest' releases to concious choice to change version, possibly in Febuary? 

Reviewer: ```suggestion
    -    Done in previous release. Dropping `latest` OASIS worker node client symlink. Sites should be making a concious choice to change version. Possibly in Febuary? 
```

Slight rewording
